tests/unit/helpers/RemoteControlTest.php

```diff
@@ -589,4 +589,76 @@ public function testGetFieldmap()
         $result = $handler->get_fieldmap($sessionKey, self::$surveyId, 'es');
         $this->assertEquals('Pregunta de ejemplo', $result[$sgq]['question']);
     }
+
+    /**
+     * Test the export_responses API call.
+     */
+    public function testExportResponses()
```

Comment:
Merge conflict from hell here. I think it's a good idea to add new tests in _new files_ instead of piling on in old files, please.

Reply:
Where was the conflict?

Reply:
https://bugs.limesurvey.org/view.php?id=18596

I took the easy way out here, @gabrieljenik , but you should be able to copy-paste the lost code in dev branch.